gpkit/nomials/nomial_math.py

```diff
@@ -633,14 +633,27 @@ def as_posyslt1(self):
         "Returns the posys <= 1 representation of this constraint."
         s = self._unsubbed.sub(self.substitutions, require_positive=False)
         posy, negy = s.posy_negy()
-        if len(negy.cs) != 1:
-            raise ValueError("SignomialInequality could not simplify to"
-                             " a PosynomialInequality")
-        else:
```

Comment:
it feels like there is some repetition with the logic that happens in `_simplify_posy_ineq`